templates/assets/env-ecs.yml

```diff
@@ -293,6 +293,25 @@ Resources:
                 log_group_name = ${AWS::StackName}
                 log_stream_name = instance/cfn-init/{instance_id}
                 datetime_format = %Y-%m-%d %H:%M:%S
+            "/consul-check.sh":
```

Comment:
Please install somewhere other than in `/`.   Suggest `/opt/mu/consul-check.sh`?